engine.py

```diff
@@ -308,7 +309,11 @@ def findBoxes(self, input, lines):
                     startchar = line[3]+2
                     endchar = 0
                 
-                if firstflag and not blackflag and whitecount > (line[3] - line[2])/2.6:
+                if (
+				       (firstflag and not blackflag and whitecount > (line[3] - line[2])/2.6)
```

Comment:
Maybe. It is difficult for me to understand, how this would change something. 

The main problem I had, was that the columns are defined with areas, but the OCR deals with boxes/words, and does not use the area definition. In the new screenshot the demand/supply number is only some pixels away from the level information. So I was capturing the number and the level in one word.